cmd/kubelet/app/options/options.go

```diff
@@ -220,6 +221,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.CgroupRoot, "cgroup-root", s.CgroupRoot, "Optional root cgroup to use for pods. This is handled by the container runtime on a best effort basis. Default: '', which means use the container runtime default.")
 	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible values: 'docker', 'rkt'. Default: 'docker'.")
 	fs.StringVar(&s.LockFilePath, "lock-file", s.LockFilePath, "<Warning: Alpha feature> The path to file for kubelet to use as a lock file.")
+	fs.BoolVar(&s.Bootstrap, "bootstrap", s.Bootstrap, "Whether kubelet should exit upon lock-file contention during bootstrap.")
```

Comment:
Wondering if it might be better to have a more specific name for this flag. `--bootstrap` is a bit generic, where maybe something along the lines of `--exit-on-lock-contention` would be more descriptive of the use/functionality.